src/Symfony/Bundle/FrameworkBundle/Resources/config/encryption.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Encryption\EncryptionInterface;
+use Symfony\Component\Encryption\Phpseclib\PhpseclibEncryption;
+use Symfony\Component\Encryption\Sodium\SodiumEncryption;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('security.encryption.sodium', SodiumEncryption::class)
+        ->set('security.encryption.phpseclib', PhpseclibEncryption::class)
+        ->alias(EncryptionInterface::class, 'security.encryption.phpseclib')
```

Comment:
shouldn't this use sodium by default when available?
since sodium is now bundled by default, that'd make more sense to me.
we should suggest somewhere to install either sodium or phpseclib when enabling the component

Reply:
We definitely should, I must have made a misstake when refactoring this.